public/components/query_set/components/query_set_form.tsx

```diff
@@ -46,6 +50,31 @@ export const QuerySetForm: React.FC<QuerySetFormProps> = ({ formState, filePicke
     handleFileContent,
   } = formState;
 
+  const downloadSampleNDJSON = async () => {
+    try {
+      const response = await fetch(SAMPLE_NDJSON_URL);
```

Comment:
Would it be possible to instead of referencing a file, could we just create the file by hand right here IN the javascript?  This would ensure that it works if you dont' have access to github (say you are in a secure environment), reduce a external dependency, and I think be really easy to do...